esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2379,7 +2429,11 @@ bool CFileSprayEx::onDespray(IEspContext &context, IEspDespray &req, IEspDespray
                 throw MakeStringException(ECLWATCH_INVALID_INPUT, "Despray %s: cannot resolve destination network IP from %s.", srcname, destip);
 
             StringBuffer destfileWithPath, umask;
+#ifdef _CONTAINERIZED
+            getDropZoneInfoByDestPlane(version, destGroup, destfile, destfileWithPath, umask);
```

Comment:
This should be used in bare metal if destPlane is specified - and not supplying it deprecated.

Reply:
getDropZoneInfoByIP  uses the environment.xml and getDropZoneInfoByIP expects the dfuserver.yaml configuration.  We can look at combining the functionality of getDropZoneInfoByIP  and getDropZoneInfoByDestPlane in 

There is some additional functionality in getDropZoneInfoByIP which hasn't been implemented in getDropZoneInfoByDestPlane.  There is a isDropZoneRestrictionEnabled configuration field for bare-metal.  

We can do this in this jira: https://track.hpccsystems.com/browse/HPCC-25593